mods/ra/rules/aircraft.yaml

```diff
@@ -434,3 +434,94 @@ U2:
 	RejectsOrders:
 	Interactable:
 	-MapEditorData:
+
+mh60:
```

Comment:
Please keep this in caps to match the other unit definitions.

Reply:
done